eng/pipelines/npm-tasks.yml

```diff
@@ -0,0 +1,31 @@
+trigger: none
+
+# This pipeline helps to run NPM admin tasks like remove or add tag to a released package version or deprecate a pacakge version
+# Following variables should be set at queue time to run this pipeline
+# variable name: TaskType
+# valid Options: 'AddTag', 'RemoveTag', 'DeprecatePackage'
+
+# variable name: PackageName
+# e.g. values: '@azure/storage-blob', '@azure/core-http'
+
+# variable name: 'PkgVersion'
+# e.g. values '1.0.0', '2.1.0.beta.1'
+
+# variable name: 'TagName'
+# e.g values: 'latest', 'next'
+
+
+jobs:
+- job: 'NPM_Admin_Job'
+
+  pool:
+    vmImage: 'windows-2019'
+
+  steps:
+    - task: PowerShell@2
+      displayName: 'Run Task'
+      inputs:
+        targetType: filePath
+        filePath: "eng/tools/npm-admin-tasks.ps1"
+        arguments: '-taskType $(TaskType) -packageName $(PackageName) -pkgVersion $(PkgVersion) -tagName $(TagName) -npmToken "$(azure-sdk-npm-token)" -reason $(Reason)'
```

Comment:
Is reason only interesting for Deprecate?

Reply:
Correct. Reason is only required to deprecate a package version